kubernetes/linux/main.sh

```diff
@@ -855,6 +855,23 @@ else
 fi
 source ~/.bashrc
 
+# manually set backpressure value using container limit only when neither backpressure or fbit tail buffer is provided through configmap
+if [ -n "${BACKPRESSURE_THRESHOLD}" ]; then
+      echo "Setting MDSD backpressure threshold from configmap"
+      export MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB=${BACKPRESSURE_THRESHOLD}
+      echo "export MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB=$MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB" >> ~/.bashrc
+      source ~/.bashrc
+elif [ -z "${FBIT_TAIL_MEM_BUF_LIMIT}" ]; then
+      if [ -n "${CONTAINER_MEMORY_LIMIT}" ]; then
+            echo "Setting MDSD backpressure threshold from container limit"
+            limit_in_mebibytes=$((CONTAINER_MEMORY_LIMIT / 1048576))
+
+            export MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB=$((limit_in_mebibytes * 50 / 100))
+            echo "export MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB=$MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB" >> ~/.bashrc
```

Comment:
add log message with MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB value . this will be helpful for any incident and also with VPA integration.

Reply:
done